zokrates_ast/src/zir/mod.rs

```diff
@@ -129,6 +166,7 @@ pub enum ZirStatement<'ast, T> {
         FormatString,
         Vec<(ConcreteType, Vec<ZirExpression<'ast, T>>)>,
     ),
+    Assembly(#[serde(borrow)] Vec<ZirAssemblyStatement<'ast, T>>),
```

Comment:
this looks weird, is there a way to put this tag somewhere else?

Reply:
yeah outside Assembly,

```
    #[serde(borrow)]
    Assembly(Vec<ZirAssemblyStatement<'ast, T>>),
 ```